contracts/bonding/BondingManager.sol

```diff
@@ -28,33 +29,54 @@ contract BondingManager is IBondingManager, Controllable {
 
     // Represents a transcoder's current state
     struct Transcoder {
-        address transcoderAddress;      // The address of this transcoder.
-        uint256 bondedAmount;           // The amount they have bonded themselves
-        uint256 delegatorWithdrawRound; // The round at which delegators to this transcoder can withdraw if this transcoder resigns
-        uint256 rewardRound;            // Last round that the transcoder called reward()
-        uint256 rewardCycle;            // Last cycle of the last round that the transcoder called reward()
-        uint8 blockRewardCut;           // Percentage of token reward that delegators pay the transcoder
-        uint8 feeShare;                 // Percentage of fees from broadcasting jobs that transcoder will share with delegators
-        uint256 pricePerSegment;        // Lowest price transcoder is willing to accept for a job. Denominated in LPT base units
-        uint8 pendingBlockRewardCut;    // Pending value for blockRewardCut to be set at the beginning of a new round
-        uint8 pendingFeeShare;          // Pending value for feeShare to be set at the beginning of a new round
-        uint256 pendingPricePerSegment; // Pending value for pricePerSegment to be set at the beginning of a new round
-        bool active;                    // Is this transcoder active. Also will be false if uninitialized
+        address transcoderAddress;                           // The address of this transcoder
+        uint256 bondedAmount;                                // The amount they have bonded themselves
+        uint256 withdrawRound;                               // The round at which delegators to this transcoder can withdraw if this transcoder resigns
+        uint256 lastRewardRound;                             // Last round that the transcoder called reward
+        uint8 blockRewardCut;                                // % of block reward cut paid to transcoder by a delegator
```

Comment:
Ended up switching back to listing `blockRewardCut`, `feeShare` and `pricePerSegment` individually. Originally wanted to switch to using `TranscodeRates` struct instead of an array to represent these fields so that we could reference each field using the struct i.e. `transcoders[msg.sender].rates.blockRewardCut`. However, Solidity does not allow the generation of public getters for structs that have struct fields. We can consider writing custom getters for the transcoder mapping if we really want to include structs a fields, but for now listing `blockRewardCut`, `feeShare` and `pricePerSegment` as separate fields allows us to keep the automatically generated getter